cmd/node/config/config.toml

```diff
@@ -580,7 +580,7 @@
 
 [VirtualMachine]
     [VirtualMachine.Execution]
-        OutOfProcessEnabled = true
+        OutOfProcessEnabled = false
```

Comment:
This would be reverted later?

Reply:
don't think so. the engine is save now to be used inprocess always